onnx/optimizer/passes/eliminate_identity.h

```diff
@@ -24,6 +24,10 @@ struct EliminateIdentity final : public PredicateBasedPass {
   }
   bool runTransform(Node* node, Graph&, NodeDestroyType& destroy_current)
       override {
+
+    if (node->output()->has_sizes()) {
```

Comment:
Do we have tests for the new logic?

Reply:
Could you elaborate, please? What should be tested against? :)